publish-packages.cmd

```diff
@@ -0,0 +1,30 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set packagesLog=publish-packages.log
+echo Running publish-packages.cmd %* > %packagesLog%
+
+if /I [%1] == [/?] goto Usage
+if /I [%1] == [/help] goto Usage
+
+REM ensure that msbuild is available
+echo Running init-tools.cmd
+call %~dp0init-tools.cmd
+
+echo msbuild.exe %~dp0src\publish.builds %* /nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog% >> %packagesLog%
```

Comment:
Could you update the script to use the `options` and `allargs` variable like the other dev workflow scripts? (i.e. build-packages.cmd)